cloud/pkg/apis/devices/v1alpha2/device_instance_types.go

```diff
@@ -38,10 +38,9 @@ type DeviceSpec struct {
 	// on edge node.
 	// +optional
 	Data DeviceData `json:"data,omitempty"`
-	// NodeSelector indicates the binding preferences between devices and nodes.
-	// Refer to k8s.io/kubernetes/pkg/apis/core NodeSelector for more details
+	// NodeName indicates the binding preferences between device and node.
 	// +optional
-	NodeSelector *v1.NodeSelector `json:"nodeSelector,omitempty"`
```

Comment:
Now one device only bind to one node, but we need to leave the interface for binding to multi-node for future. We can have more discussion.

Reply:
Yes, we can discuss it later.